bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -264,6 +264,11 @@ The channels are kept consistent as much as possible from brand to brand to make
 | `tooDarkAlarm` | Switch (read only) | ONVIF cameras only will reflect the status of the ONVIF event of the same name. |
 | `pollImage` | Switch | This control can be used to manually start and stop using your CPU to create snapshots from a RTSP source. If you have a snapshot URL setup in the binding, only then can this control can be used to update the Image channel. |
 | `zoom` | Dimmer | Works with ONVIF cameras that can be moved. |
+| `acceptedCardNumber` | String (read only) | This channel shows the last Pass Card Number, authorized by Dahua VTO to open door. This could be used by Openhab's presence detection and security monitoring. Cards could be programmed into VTO without VTH with a help of Dahua SmartPSS soft <https://www.youtube.com/watch?v=qzR_ILT7j-0>. After that cards appear in VTO's web interface. The channel doesn't show rejected/unauthorised cards. |
```

Comment:
```suggestion
| `acceptedCardNumber` | String (read only) | This channel shows the last accepted access card number that opened the door. The channel doesn't show rejected/unauthorized cards. |
```
1. You spelled openHAB wrongly as we are picky about the capitals in our docs.
2. The channel will get re-used by other brands most likely in the future so removing mention of brands and keeping this short I think is a better idea. 
3. I probably would prefer to call the channel `lastAcceptedCard` as I would have to check how other brands that have a similar feature work. I suspect some may use a card hash ID, some may use a users name, and some may use a index number.